apps/system/utils/fscmd.c

```diff
@@ -282,6 +282,10 @@ static int tash_cat(int argc, char **args)
 		FSCMD_OUTPUT(MISSING_ARGS FSCMD_CAT_USAGE, args[0]);
 		return ERROR;
 	} else if (argc == 2) {
+		if (!strncmp(args[1], "--help", strlen("--help") + 1)){
```

Comment:
Why to check for terminating null character?

Reply:
Let's assume args[1] = "--helpYYYzwmj0k", if there is no check for terminating null character, it would be considered as correct input. But it't not